ouroboros-consensus/src/Ouroboros/Consensus/Network/NodeToClient.hs

```diff
@@ -4,7 +4,6 @@
 {-# LANGUAGE QuantifiedConstraints #-}
 {-# LANGUAGE RecordWildCards       #-}
 {-# LANGUAGE ScopedTypeVariables   #-}
-{-# LANGUAGE TypeApplications      #-}
```

Comment:
Nice catch! I'm always impressed when people notice these things. Very thorough.

Reply:
hls found it, I shouldn't take the credits :)